python/mysat1_telemetry.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# 
+# Copyright 2018 Daniel Estevez <daniel@destevez.net>
+# 
+# This is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+# 
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with this software; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+# 
+
+from construct import *
+from adapters import *
+
+Timestamp = UNIXTimestampAdapter(Int32ub)
+
+Callsign = Struct(
+    'callsign' /  Int8ub[5])
```

Comment:
It would be better to use `Bytes(5)` instead of `Int8ub[5]`.

Reply:
ok, will do